aiohttp/streams.py

```diff
@@ -580,9 +579,8 @@ def set_exception(self, exc: BaseException) -> None:
             self._waiter = None
             set_exception(waiter, exc)
 
-    def feed_data(self, data: _T, size: int = 0) -> None:
-        self._size += size
```

Comment:
Why is this removed? This will surely break the limit functionality. Just use `len(data)` to calculate the actual size?

Reply:
And if we don't delete this line, the `TestFlowControlDataQueue.test_resume_on_read` test will fall